.github/workflows/Nightly_Func_Env_CI.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         RUNNER_PATH: ${{ secrets.RUNNER_PATH }}
       run: |
         echo '>>>>>>>>>>>>>>>>>>>>>>>>>> Remove image on provision  >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>'
-        echo "if [[ \"\$(sudo podman images -q quay.io/ebattat/benchmark-runner 2> /dev/null)\" != \"\" ]]; then sudo podman rmi -f \$(sudo podman images -q quay.io/ebattat/benchmark-runner 2> /dev/null); fi" > "$RUNNER_PATH/remove_image.sh"
+        echo "if [[ \"\$(sudo podman images -q ${{ secrets.QUAY_ROBOT_BENCHMARK_RUNNER_PASSWORD }} 2> /dev/null)\" != \"\" ]]; then sudo podman rmi -f \$(sudo podman images -q ${{ secrets.QUAY_ROBOT_BENCHMARK_RUNNER_PASSWORD }} 2> /dev/null); fi" > "$RUNNER_PATH/remove_image.sh"
```

Comment:
This doesn't look right -- shouldn't this be QUAY_BENCHMARK_RUNNER_REPOSITORY rather than QUAY_BENCHMARK_RUNNER_PASSWORD?  The entire diff looks like this.